src/modules/jcms_rest/tests/src/EndpointValidatorTest.php

```diff
@@ -35,30 +37,29 @@ public function setUp() {
         require_once($this->projectRoot . '/.puli/GeneratedPuliFactory.php');
       }
     }
+    // Setup Puli.
+    $this->resourceRepository = (new GeneratedPuliFactory)->createRepository();
+    /*$factoryClass = PULI_FACTORY_CLASS;
+    $this->resourceRepository = (new $factoryClass())->createRepository();*/
```

Comment:
We don't need it at the moment but it was left in so we could use it later if the situation changes as the current implementation is hacky. Would you like it removed?